LDM-148.tex

```diff
@@ -414,50 +405,73 @@ \subsubsection{Image and EFD Archiving}\label{image-and-efd-archiving}
+It maintains a higher level of service availability than the Data Backbone and so provides a staging area for to-be-archived data prior to ingestion into the Data Backbone.
+After images and metadata are ingested, they remain available through the OODS for a policy-configured amount of time.
 
-This service receives commands from the OCS and invokes the OCS Driven
-Batch Processing service in the NCSA Enclave Level 1 Domain to execute
-corresponding science payloads. It is used for modest-latency analysis
+\subsubsection{OCS Driven Batch}\label{ocs-driven-batch}
+
+This service receives commands from the OCS and invokes a Batch Computing
+service to execute configured science payloads.  The service can be
+configured to execute on the Commissioning Cluster at the Base or in the
+General Production Enclave at NCSA.
+It is used for modest-latency analysis
 of images during Commissioning and for processing daily calibration
-images in normal observing operations. A summary status for the
-processing performed is returned to the OCS for each command, following
-the normal OCS commanding protocol.
+images in normal observing operations.  Images and metadata are taken
+from the Data Backbone, and results are provided back to the Data
+Backbone; there is no direct connection from this service to the Camera
+Data System. This obviously bounds the minimum latency from image
+acquisition to processing start by the latency of the Archiving service
+and Data Backbone transfer. A summary status for the processing
+performed is sent to the OCS Driven Batch Control service to be returned
+to the OCS.
 
 \subsubsection{Telemetry Gateway}\label{telemetry-gateway}
 
-This service obtains information from the NCSA Enclave Level 1 Domain,
+This service obtains information from the NCSA L1 Enclave,
 particularly status and quality metrics from Prompt Processing of images
-and the Level 1 Quality Control service, and transmits it to the OCS as
-specified in the Data Management-OCS Software Communication Interface
+and the Prompt Quality Control service, and transmits it to the OCS as
+specified in the ``Data Management-OCS Software Communication Interface''
 (\citeds{LSE-72}). Note that more detailed information on the status and
 performance of DMS services will also be available to Observatory
 operators through remote displays originated from the
-Management/Monitoring infrastructure services in all DMS computational
+Service Management/Monitoring infrastructure services in all DMS computational
 environments.
 
 \subsection{Interfaces}\label{base-interfaces}
 
-OCS to all Base Enclave services: these interface through the SAL
+OCS to all Base L1 Enclave services: these interface through the SAL
 library provided by the OCS subsystem.
 
-Archiver and Catch-Up Archiver to Data Backbone: files are copied to
+Archiver and Catch-Up Archiver to Observatory Operations Data Service:
```

Comment:
Add `(OODS)` here to indicate that that is what later usage of OODS means.

Reply:
The abbreviation is defined above in line 395 and has already been used twice by this point.